### PR TITLE
update imagemagick and rebuild dependencies sufficient to get necessary libtool files for the build

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=imagemagick crew update
```

### CODE REVIEW

File: manifest/armv7l/i/imagemagick7.filelist [hunk 1]
```diff
@@ -450,14 +449,14 @@
...
-/usr/local/lib/perl5/5.38.1/vendor_perl/auto/Image/Magick/Q16HDRI/autosplit.ix
-/usr/local/lib/perl5/5.38.1/vendor_perl/auto/Image/Magick/autosplit.ix
+/usr/local/lib/perl5/armv8l-linux-thread-multi/Image/Magick.pm
+/usr/local/lib/perl5/armv8l-linux-thread-multi/Image/Magick/Q16HDRI.pm
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/.packlist
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/Magick.so
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/Q16HDRI/Q16HDRI.so
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/Q16HDRI/autosplit.ix
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/autosplit.ix
+/usr/local/lib/perl5/armv8l-linux-thread-multi/perllocal.pod
```
Comment (uberhacker) [inline manifest/armv7l/i/imagemagick7.filelist:459]: This path name does not match the architecture.

Comment (satmandu) [reply manifest/armv7l/i/imagemagick7.filelist:459]: Can you see if it works on arm machines?

Armv8l is what armv7l reports when loaded on an aarch64 kernel.

Comment (satmandu) [reply manifest/armv7l/i/imagemagick7.filelist:459]: Also the entire `imagemagick7` autotools build is ornery and basically impossible to debug, so I don't know how to fix this short of building it in a qemu container which might take a long time...

Comment (uberhacker) [reply manifest/armv7l/i/imagemagick7.filelist:459]: I'll just merge it and we can fix if it breaks.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**